mobile_insight/analyzer/vr_test_analyzer.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python

```

Comment:
@happyqiaomai If this analyzer is for test only, please move it to separate testing folder